framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1453,15 +1453,27 @@ public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree lhsTree) {
     /**
      * Returns the type of a varargs array of a method invocation or a constructor invocation.
      *
+     * <p>A null return value might mean:
+     *
+     * <ul>
+     *   <li>This type-checker is not using dataflow analysis.
+     *   <li>The given tree is dead code, so dataflow analysis never reached it to refine its type.
```

Comment:
This method is only using the `Node` to get the synthetic tree.  The returned type is computed using the tree, not the dataflow value.